### ISSUE TITLE
Is this project still live ?

### ISSUE THREAD

**Issue by maciey** [jzoss/Git-Source-Control-Provider#122]
I love this plugin but cant see any new commit since ~2yrs

**Comment by jzoss** [comment]
Yes.. I still try and patch it when things come up.. I've been a bit busy
as of late.. But totally still updating it. Also.. totally looking for help
if anyone is bored!

**Comment by sukai-cheng** [comment]
Hi there,  is this project still live ?
